lib/kitchen/provisioner/chef_infra.rb

```diff
@@ -0,0 +1,167 @@
+#
+# Author:: Fletcher Nichol (<fnichol@nichol.ca>)
+#
+# Copyright (C) 2013, Fletcher Nichol
```

Comment:
We should add an additional copyright line here, `Copyright (C) Chef Software Inc.`